library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/AllowAllWireframeMapperTest.kt

```diff
@@ -179,4 +182,32 @@ internal class AllowAllWireframeMapperTest {
     fun `M return the ImageMapper W getImageMapper`() {
         assertThat(testedAllowAllWireframeMapper.imageMapper).isEqualTo(mockImageWireframeMapper)
     }
+
+    @Test
+    fun `M delegate to DecorViewWireframeMapper W map { DecorView }`() {
+        // Given
+        val mockView: View = mockViewWithNoViewTypeParent()
+        whenever(mockDecorViewWireframeMapper.map(mockView, fakeSystemInformation))
+            .thenReturn(mockDecorViewWireframes)
+
+        // When
+        val wireframes = testedAllowAllWireframeMapper.map(mockView, fakeSystemInformation)
+
+        // Then
+        assertThat(wireframes).isEqualTo(mockDecorViewWireframes)
+    }
+
+    @Test
+    fun `M delegate to DecorViewWireframeMapper W map { view with parent as no View type }`() {
+        // Given
+        val mockView: View = mockViewWithEmptyParent()
```

Comment:
should it be `mockViewWithNoViewTypeParent` here instead? 

```suggestion
        val mockView: View = mockViewWithEmptyParent()
```